app/src/main/res/layout-land/hints_summary.xml

```diff
@@ -103,12 +99,10 @@
 
       <TextView
         android:id="@+id/hints_and_solution_summary"
+        style="@style/Body"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
```

Comment:
Remove `layout_width` and `layout_height` attribute as it is already mentioned in `Body` style. (Make sure its `wrap_content` only in styles.xml)